WordPress/gutenberg
# Apply elevation scale in components package

<img width="976" alt="Elevation" src="https://github.com/user-attachments/assets/deebfabe-b90a-426c-a69f-09f47940681c">



Follow up to https://github.com/WordPress/gutenberg/pull/64108

## What?
This PR applies the recently-added elevation scale to components in the core package.

> [!IMPORTANT]  
> This PR gives visibility to new steps in the elevation scale for the first time, most notably `medium` elevation applied to menus / popovers. If we're unhappy with the appearance, or even where certain components are placed, now would be a good time to adjust. For me, `medium` elevation probably needs to be toned down a bit.

## Why?
See https://github.com/WordPress/gutenberg/issues/63757

## Updated components

### Custom Select Control v2
Applied `medium` elevation

| Before | After |
| --- | --- |
| <img width="480" alt="Screenshot 2024-09-09 at 16 46 11" src="https://github.com/user-attachments/assets/05f7eaf0-b762-43f5-b8da-60e16af9f344"> | <img width="571" alt="Screenshot 2024-09-17 at 09 54 26" src="https://github.com/user-attachments/assets/3c85ffba-e9a8-4a96-809c-0b2a002a5e21"> |




### DropdownMenu v2
Applied `medium` elevation

| Before | After |
| --- | --- |
| <img width="343" alt="Screenshot 2024-09-09 at 16 47 32" src="https://github.com/user-attachments/assets/f56ecb83-a452-403f-b777-3cf3df63235a"> | <img width="353" alt="Screenshot 2024-09-17 at 09 55 12" src="https://github.com/user-attachments/assets/3417e2f4-74ac-4f50-978b-a6b981a683c3"> |



### Form Toggle
Applied `x-small` elevation to handle

| Before | After |
| --- | --- |
| <img width="66" alt="Screenshot 2024-09-09 at 16 48 56" src="https://github.com/user-attachments/assets/ca9be2ad-6c25-484f-ad37-9ec968598afd"> | <img width="62" alt="Screenshot 2024-09-09 at 16 48 59" src="https://github.com/user-attachments/assets/d426db7c-635b-4f82-9d8e-cad44c500c4c"> |


### Popover
Applied `medium` elevation

| Before | After |
| --- | --- |
| <img width="351" alt="Screenshot 2024-09-09 at 16 50 20" src="https://github.com/user-attachments/assets/93ece823-3961-409d-9290-c30d3fe7c770"> | <img width="347" alt="Screenshot 2024-09-17 at 09 55 46" src="https://github.com/user-attachments/assets/2c875082-0de0-4b95-b7da-dd30965d240f"> |



### Range Control
Applied `x-small` elevation to handle

| Before | After |
| --- | --- |
| <img width="343" alt="Screenshot 2024-09-09 at 16 51 23" src="https://github.com/user-attachments/assets/f25ea5ac-0d88-469e-a7ae-e2476c743252"> | <img width="378" alt="Screenshot 2024-09-09 at 16 51 29" src="https://github.com/user-attachments/assets/6abbc797-e40e-43b7-8141-432512a0bfc0"> |

### Resizeable Box
Applied `x-small` elevation to handles

| Before | After |
| --- | --- |
| <img width="427" alt="Screenshot 2024-09-09 at 16 52 16" src="https://github.com/user-attachments/assets/919322b9-0f4b-4320-96ae-1962ee088d0e"> | <img width="445" alt="Screenshot 2024-09-17 at 09 56 17" src="https://github.com/user-attachments/assets/04b1b9bf-1863-4250-aaf1-4708c971b5a5"> |


### Snackbar
Applied `small` elevation

| Before | After |
| --- | --- |
| <img width="417" alt="Screenshot 2024-09-09 at 16 53 13" src="https://github.com/user-attachments/assets/b1121f7f-64ca-4303-89fe-36345ec0c84c"> | <img width="579" alt="Screenshot 2024-09-12 at 22 48 33" src="https://github.com/user-attachments/assets/ed765570-3a82-40f8-b352-dd5bcde58f62"> |


### Tooltip
Applied `small` elevation

| Before | After |
| --- | --- |
| <img width="144" alt="Screenshot 2024-09-09 at 16 54 13" src="https://github.com/user-attachments/assets/97bbb7e8-43c1-4638-9614-7d0f6e2d1105"> | <img width="169" alt="Screenshot 2024-09-12 at 22 49 24" src="https://github.com/user-attachments/assets/5a3ab91c-ba86-403c-a8c6-dc5ac62c8a25"> |





## Testing Instructions
1. `npm run storybook:dev`
2. Check the components detailed above appear as expected
3. Test the Editor

> [!NOTE]  
> This PR does not touch the `Elevation` component. So far as I can tell it is not frequently used – it might be good to discuss it's long-term future.
